### ISSUE TITLE
Conversion from OpenAPI 3.0 to Swagger 2.0 fails if "components" is missing

### ISSUE THREAD

**Issue by mkauf** [LucyBot-Inc/api-spec-converter#147]
I've tried to convert an OpenAPI 3.0 specification to Swagger 2.0:
```
api-spec-converter -f openapi_3 -t swagger_2 test.json
```

test.json:
```
{
  "openapi": "3.0.0",
  "info": {
    "title": "test",
    "version": "1.0"
  },
  "paths": {
    "/": {
      "get": {
        "responses": {
          "default": {
            "description": "test"
          }
        }
      }
    }
  }
}
```

The conversion fails:
```
Fatal TypeError: Cannot read property 'securitySchemes' of undefined
    at convertSecurityDefinitions (/api-spec-converter/lib/converters/openapi3_to_swagger2.js:194:62)
    at convert (/api-spec-converter/lib/converters/openapi3_to_swagger2.js:18:3)
    at converters.swagger_2.Promise.method.oa (/api-spec-converter/lib/formats/openapi_3.js:17:26)
    at tryCatcher (/api-spec-converter/node_modules/bluebird/js/release/util.js:16:23)
    at /api-spec-converter/node_modules/bluebird/js/release/method.js:15:34
    at Promise.try (/api-spec-converter/lib/base_format.js:162:12)
    at tryCatcher (/api-spec-converter/node_modules/bluebird/js/release/util.js:16:23)
    at Function.Promise.attempt.Promise.try (/api-spec-converter/node_modules/bluebird/js/release/method.js:39:29)
    at module.exports.BaseFormat.convertTo (/api-spec-converter/lib/base_format.js:154:21)
    at Converter.getSpec.then.fromSpec (/api-spec-converter/index.js:33:32)
    at tryCatcher (/api-spec-converter/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/api-spec-converter/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/api-spec-converter/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/api-spec-converter/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/api-spec-converter/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/api-spec-converter/node_modules/bluebird/js/release/async.js:133:16)
```

Workaround: Add an empty "components" property, now the conversion succeeds:
```
{
  "openapi": "3.0.0",
  "info": {
    "title": "Minimal specification",
    "version": "1.0"
  },
  "paths": {
    "/": {
      "get": {
        "responses": {
          "default": {
            "description": "test"
          }
        }
      }
    }
  },
  "components": {
  }
}
```

**Comment by bobby-brennan** [comment]
Thanks for the report, this should be fixed in 2.6.2

**Comment by mkauf** [comment]
It's fixed, thanks!
